docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/article/paragraph--body-section.html.twig

```diff
@@ -9,7 +9,8 @@
  */
 #}
 {# Setup attribs if schema org is enabled #}
-{% set section_attribs = create_attribute() %}
+{% set section_attribs = create_attribute().addClass('usa-prose') %}
+
```

Comment:
`usa-prose` should not wrap the h2. You should just be SMARTLY wrapping:
```
  {% if schema_org_type %}
    <div itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
      <div itemprop="text">
  {% endif %}
  {{ content.field_body_section_content }}
  {% if schema_org_type %}
      </div>
    </div>
  {% endif %}
```

Smartly means that if there is `<div itemprop="text">` then just add a class to that div. If it is not then you need to add a div when it is not schema org. 

THEN you need to add `nci-heading-h2` to the h2. Or, make it `cgdp-article-body__heading` and extend `nci-heading-h2`.  Then you can also control the `<p>` tag spacing in `cgdp-article-body__heading` rather than do it in a hacky.